pyproject.toml

```diff
@@ -91,8 +91,8 @@ graph_func = ['pywhy-graphs']
 viz = ['pygraphviz']
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
```

Comment:
What's the motivation for this change?

Reply:
According to [their docs](https://github.com/mtkennerly/poetry-dynamic-versioning), this is the recommended configuration for the dynamic-versioning plugin in 1.2.0

Reply:
It looks like swapping the build-backend caused the sdist checker to fail. Reverting to pip-based install.